package/Dockerfile

```diff
@@ -64,6 +65,10 @@ RUN curl -sLf https://github.com/rancher/machine/releases/download/${CATTLE_MACH
     unzip docker-machine-driver-linode_linux-amd64.zip -d /opt/drivers/management-state/bin && \
     rm docker-machine-driver-linode_linux-amd64.zip
 
+RUN curl -LO https://releases.rancher.com/harvester-node-driver/${DOCKER_MACHINE_HARVESTER_VERSION}/docker-machine-driver-harvester-amd64.tar.gz && \
+    tar -xf docker-machine-driver-harvester-amd64.tar.gz -C /opt/drivers/management-state/bin && \
+    rm docker-machine-driver-harvester-amd64.tar.gz
+
```

Comment:
Is there are reason to download this here? Since the URL is passed in `machinedriver_data.go`, then it will be downloaded there when necessary.

The answer here is probably "air gap" but I want to make sure I understand what is happening here before approving.

Reply:
Hey @thedadams  you are absolutely correct. The reason for downloading it is for air gap environment.